src/chatd.cpp

```diff
@@ -3815,7 +3833,15 @@ Idx Chat::msgConfirm(Id msgxid, Id msgid)
         CHATD_LOG_DEBUG("msgConfirm: updated opcode MSGUPDx to MSGUPD and the msgxid=%u to msgid=%u of %d message/s in the sending queue", msgxid, msgid, count);
     }
 
-    CALL_LISTENER(onMessageConfirmed, msgxid, *msg, idx);
+    bool tsUpdated = false;
+    if (timestamp != 0)
+    {
+        msg->ts = timestamp;
```

Comment:
The `ts` is updated in RAM, but not in DB

Reply:
Solved it